openapi.yml

```diff
@@ -237,63 +237,24 @@ components:
       required:
         - hasAdminPolicy
     AdminPolicyAccessTemplate:
-      description: 'Provides the template of access settings that is copied to the items goverend by an AdminPolicy. This is almost the same as DROAccess, but it provides no defaults and has no embargo.'
+      description: 'Provides the template of access settings that is copied to the items governed by an AdminPolicy. This is almost the same as DROAccess, but it provides no defaults and has no embargo.'
       type: object
       additionalProperties: false
+      oneOf:
+        - $ref: '#/components/schemas/DarkAccess'
+        - $ref: '#/components/schemas/CitationOnlyAccess'
+        - $ref: '#/components/schemas/ControlledDigitalLendingAccess'
+        - $ref: '#/components/schemas/LocationBasedAccess'
+        - $ref: '#/components/schemas/LocationBasedDownloadAccess'
+        - $ref: '#/components/schemas/StanfordAccess'
+        - $ref: '#/components/schemas/WorldAccess'
       properties:
-        view:
-          type: string
-          enum:
-            - 'world'
-            - 'stanford'
-            - 'location-based'
-            - 'citation-only'
-            - 'dark'
-        controlledDigitalLending:
-          description: Available for controlled digital lending.
-          type: boolean
         copyright:
-          description: The human readable copyright statement that applies
-          example: Copyright World Trade Organization
-          type: string
-          nullable: true
-        download:
-          description: >
-            Download access level. This is used in the transition from Fedora as
-            a way to set a default download level at registration that is copied
-            down to all the files.
-
-          type: string
-          enum:
-            - 'world'
-            - 'stanford'
-            - 'location-based'
-            - 'none'
-        location:
-          description: >
-            If access or download is "location-based", this indicates which
-            location should have access. This is used in the transition from
-            Fedora as a way to set a default location at registration that is
-            copied down to all the files.
-
-          type: string
-          nullable: true
-          enum:
-            - 'spec'
-            - 'music'
-            - 'ars'
-            - 'art'
-            - 'hoover'
-            - 'm&m'
+          $ref: "#/components/schemas/Copyright"
```

Comment:
Instead of having identical copyright, license, and use statement properties throughout the spec, create new schemas for each and use them.